hack/verify-generated.sh

```diff
@@ -2,16 +2,22 @@
 set -euo pipefail
 
 function print_failure {
-  echo "There are unexpected changes to the tree when running 'make generate' and `make manifests`. Please"
+  echo "There are unexpected changes to the tree when running 'make generate manifests olm-manifests validate-bundle'. Please"
   echo "run these commands locally and double-check the Git repository for unexpected changes which may"
   echo "need to be committed."
   exit 1
 }
 
 if [ "${OPENSHIFT_CI:-false}" = true ]; then
+  echo "> generating the manifests"
   make generate
   make manifests
+  echo "> generating the OLM bundle"
+  make olm-manifests
+  test -z "$(git status --porcelain | \grep -v '^??')" || print_failure
 
-  test -z "$(git status --porcelain)" || print_failure
-  echo "verified generated manifests and deep copy"
+  echo "> verifying the OLM bundle"
+  make validate-bundle || print_failure
```

Comment:
Instead of calling validate-bundle in here can you add it as a prereq for the `verify` target in the _Makefile_.

Reply:
Moved the OLM part into a dedicated file and added it to `verify` target.